src/util.js

```diff
@@ -0,0 +1,45 @@
+/**
```

Comment:
Yes, I think `types.js` is good. In general my rule of thumb is this: if you want to name something "util" or "common", first try the name "dump" - if it still works then go ahead :)